### PR TITLE
phonenumber in product table

### PR DESCRIPTION
**Pull request by erkinun**
# Description

Phone numbers will be rendered when available in Product table.

Screenshot 2021-03-30 at 11 46 09
Screenshot 2021-03-30 at 11 45 57

# Checklist

Pull request contains:

- [ ] A new component
- [x] Component maintenance: improvement / bug fix / etc
- [x] Component library change: storybook / webpack / etc

Definition of done:

- [ ] Includes theme changes for both Uswitch and money
- [ ] Work has been tested in multiple browsers
- [x] PR description includes description of change
- [x] PR description includes screenshot of change
- [ ] If new component, designer has approved screenshot
- [ ] If the change will affect other teams, that team knows about this change
- [x] If introducing a new component behaviour, added a story to cover that case.

### CODE REVIEW

**Comment by jonathan-fielding** [comment]
Spacing above the phone number looks off

File: src/compounds/legacy-product-table/src/index.tsx [hunk 1]
```diff
@@ -557,6 +557,45 @@ const Eligibility: React.FC<EligibilityProps> = ({
...
+    theme: { name, colors = {} }
+  }: any = useThemeUI()
+
+  const iconColor = name === 'Money' ? colors.fuschia : colors.primary
+
+  return (
+    <div
+      sx={{
+        mr: '10px',
+        fontWeight: 600,
```
Comment (jonathan-fielding) [inline src/compounds/legacy-product-table/src/index.tsx:575]: this is hard coded, you can use values from the theme

File: src/compounds/legacy-product-table/src/index.tsx [hunk 2]
```diff
@@ -557,6 +557,45 @@ const Eligibility: React.FC<EligibilityProps> = ({
...
+        fontFamily: 'Varela Round,Open Sans,Helvetica Neue,Arial,sans-serif',
```
Comment (jonathan-fielding) [inline src/compounds/legacy-product-table/src/index.tsx:576]: fonts should be based on theme

File: src/compounds/legacy-product-table/src/index.tsx [hunk 3]
```diff
@@ -557,6 +557,45 @@ const Eligibility: React.FC<EligibilityProps> = ({
...
+        textAlign: 'right'
+      }}
+    >
+      <span sx={{ mr: '5px', stroke: iconColor }}>
+        <Icon glyph="phone" color={iconColor} size={24} />
+      </span>
+      <span
+        sx={{
+          color: colors.text,
+          padding: '4px'
```
Comment (jonathan-fielding) [inline src/compounds/legacy-product-table/src/index.tsx:590]: is there a matching theme value

File: src/compounds/legacy-product-table/src/index.tsx [hunk 4]
```diff
@@ -609,21 +650,31 @@ const LegacyProductTable: React.FC<LegacyProductTableProps> = ({
...
         {badge && <Badge text={badge} />}
 
-        <header>
+        <header
+          sx={{
+            display: ['flex'],
+            justifyContent: ['space-between'],
+            alignItems: 'center',
+            fontSize: '18px',
+            fontFamily: 'Varela Round,Arial,sans-serif',
```
Comment (jonathan-fielding) [inline src/compounds/legacy-product-table/src/index.tsx:659]: fonts come from theme

File: src/compounds/legacy-product-table/src/index.tsx [hunk 5]
```diff
@@ -609,21 +650,31 @@ const LegacyProductTable: React.FC<LegacyProductTableProps> = ({
       >
         {badge && <Badge text={badge} />}
 
-        <header>
+        <header
+          sx={{
+            display: ['flex'],
+            justifyContent: ['space-between'],
+            alignItems: 'center',
+            fontSize: '18px',
```
Comment (jonathan-fielding) [inline src/compounds/legacy-product-table/src/index.tsx:658]: can we select this from theme

File: src/compounds/legacy-product-table/src/index.tsx (hunk 3 above)
Comment (erkinun) [reply src/compounds/legacy-product-table/src/index.tsx:590]: I think I can use `theme.space.xxxs`, does it make sense?

File: src/compounds/legacy-product-table/src/index.tsx [hunk 6]
```diff
@@ -557,6 +557,45 @@ const Eligibility: React.FC<EligibilityProps> = ({
   )
 }
 
+interface TelephoneInfoProps {
+  telephone?: string
```
Comment (samhstn) [inline src/compounds/legacy-product-table/src/index.tsx:561]:
```suggestion
  telephone: string
```

We only render if defined below

File: src/compounds/legacy-product-table/src/index.tsx [hunk 7]
```diff
@@ -562,6 +562,47 @@ const Eligibility: React.FC<EligibilityProps> = ({
...
+  }: any = useThemeUI()
+
+  const iconColor = name === 'Money' ? colors.fuschia : colors.primary
+
+  return (
+    <div
+      sx={{
+        mr: '10px',
+        fontWeight: fontWeights.xbold,
+        fontFamily: fonts.body,
```
Comment (samhstn) [inline src/compounds/legacy-product-table/src/index.tsx:581]:
```suggestion
        fontFamily: 'xbold',
        fontFamily: 'body',
```

Might not need to use `useThemeUI`

File: src/compounds/legacy-product-table/src/index.tsx [hunk 8]
```diff
@@ -562,6 +562,47 @@ const Eligibility: React.FC<EligibilityProps> = ({
   )
 }
 
+interface TelephoneInfoProps {
+  telephone?: string
```
Comment (samhstn) [inline src/compounds/legacy-product-table/src/index.tsx:566]: 🚩

File: src/compounds/legacy-product-table/src/index.tsx [hunk 9]
```diff
@@ -562,6 +562,47 @@ const Eligibility: React.FC<EligibilityProps> = ({
...
+}
+
+const TelephoneInfo: React.FC<TelephoneInfoProps> = ({ telephone }) => {
+  const {
```
Comment (jonathan-fielding) [inline src/compounds/legacy-product-table/src/index.tsx:570]: unccessary, lets pair on this bit tommorow

**Review by samhstn — CHANGES_REQUESTED** [review]
Let me know if you need a hand with `useThemeUI`

File: src/compounds/legacy-product-table/src/index.tsx [hunk 10]
```diff
@@ -562,6 +562,47 @@ const Eligibility: React.FC<EligibilityProps> = ({
...
+    <div
+      sx={{
+        mr: '10px',
+        fontWeight: fontWeights.xbold,
+        fontFamily: fonts.body,
+        verticalAlign: 'middle',
+        display: 'flex',
+        height: '24px',
+        lineHeight: '24px',
+        '@media (max-width: 500px)': {
```
Comment (jonathan-fielding) [inline src/compounds/legacy-product-table/src/index.tsx:586]: use array syntax for applying breakpoints, dont use media queries here

File: src/compounds/legacy-product-table/src/index.tsx (hunk 8 above)
Comment (erkinun) [reply src/compounds/legacy-product-table/src/index.tsx:566]: like the flat emoji!

**Review by samhstn — APPROVED** [review]
Nice work

**Review by jonathan-fielding — DISMISSED** [review]

**Review by jonathan-fielding — APPROVED** [review]

**Review by samhstn — APPROVED** [review]

**Pull request merged**